docs/tutorials/interfaces.rst

```diff
@@ -0,0 +1,52 @@
+.. _interfaces:
+========================
+Interfaces
+========================
+
+
+One of the biggest strengths of Orquestra is its modularity. Integrating new backends, optimizers, ansatzes, etc. does not require changing the core code - it requires only creating a new module that conforms to existing interfaces and therefore can be used across the whole platform. These interfaces allow us to implement various modules and functions.
```

Comment:
"These interfaces allow us to implement various modules and functions" This feels kinda redundant.

Reply:
Removed